src/components/PersonLink.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+interface Props {
+  person: Person | undefined;
```

Comment:
you render this component only when the person exists
```suggestion
  person: Person;
```